docs/quickstart-aks.md

```diff
@@ -85,11 +85,9 @@ BKPR releases are available for 64-bit versions of Linux, macOS and Windows plat
 For convenience lets define a environment variable with the BKPR version:
 
 ```bash
-export BKPR_VERSION=vX.Y.Z
+export BKPR_VERSION=$(curl --silent "https://api.github.com/repos/bitnami/kube-prod-runtime/releases/latest" | sed -n 's/.*"tag_name": *"\([^"]*\)".*/\1/p')
```

Comment:
since the guide lists `jq` as a requirement, we could use `jq -r '.tag_name'` instead of the `sed` command.